components/library/gdk-pixbuf/Makefile

```diff
@@ -84,12 +84,6 @@ COMPONENT_TEST_TRANSFORMS += '-ne "/^XFAIL/p" '
 COMPONENT_TEST_TRANSFORMS += '-ne "/^ERROR/p" '
 COMPONENT_TEST_TRANSFORMS += '-ne "/^\# /p" '
 
-build: $(BUILD_32_and_64)
-
-install: $(INSTALL_32_and_64)
-
-test: $(TEST_32_and_64)
-
 # Auto-generated dependencies
 REQUIRED_PACKAGES += image/library/libjpeg8-turbo
 REQUIRED_PACKAGES += image/library/libpng16
```

Comment:
/usr/gcc/7/lib/gcc/i386-pc-solaris2.11/7.5.0/include/stddef.h:435: syntax error, unexpected identifier in '  __float128 __max_align_f128 __attribute__((__aligned__(__alignof(__float128))));' at '__float128'
In file included from /usr/lib/amd64/glib-2.0/include/glibconfig.h:9:0,
                 from /usr/include/glib-2.0/glib/gtypes.h:32,
                 from /usr/include/glib-2.0/glib/galloca.h:32,
                 from /usr/include/glib-2.0/glib.h:30,
                 from /export/home/andreas/oi-userland/components/library/gdk-pixbuf/build/i86/gdk-pixbuf/tmp-introspectv_yd1ewm/GdkPixbuf-2.0.c:2:
/usr/include/glib-2.0/glib/gtypes.h: In function '_GLIB_CHECKED_ADD_U64':
/usr/include/glib-2.0/glib/gmacros.h:738:31: error: static assertion failed: "Expression evaluates to false"
 #define G_STATIC_ASSERT(expr) _Static_assert (expr, "Expression evaluates to false")
                               ^
/usr/include/glib-2.0/glib/gtypes.h:466:3: note: in expansion of macro 'G_STATIC_ASSERT'
   G_STATIC_ASSERT(sizeof (unsigned long long) == sizeof (guint64));
   ^~~~~~~~~~~~~~~
Traceback (most recent call last):
  File "/usr/lib/python3.5/distutils/unixccompiler.py", line 118, in _compile
    extra_postargs)
  File "/usr/lib/python3.5/distutils/ccompiler.py", line 909, in spawn
    spawn(cmd, dry_run=self.dry_run)
  File "/usr/lib/python3.5/distutils/spawn.py", line 36, in spawn
    _spawn_posix(cmd, search_path, dry_run=dry_run)
  File "/usr/lib/python3.5/distutils/spawn.py", line 159, in _spawn_posix
    % (cmd, exit_status))
distutils.errors.DistutilsExecError: command '/usr/gcc/7/bin/gcc' failed with exit status 1

Reply:
This seems like we are missing something in gcc-7.

Reply:
@AndWac All these errors are because 32-bit gobject-introspection was dropped in 2020. Solution is to move all the dependencies to 64-bit only and drop 32-bit.